libs/types/src/tokens.rs

```diff
@@ -204,6 +204,48 @@ pub struct CustomMetadata {
 
 	/// Whether an asset can be used as a currency to fund Centrifuge Pools.
 	pub pool_currency: bool,
+
+	/// The ways this token is cross-chain transferable
+	pub transferability: CrossChainTransferability,
+}
+
+#[derive(
+	Clone,
+	Copy,
+	Default,
+	PartialOrd,
+	Ord,
+	PartialEq,
+	Eq,
+	Debug,
+	Encode,
+	Decode,
+	TypeInfo,
+	MaxEncodedLen,
+)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum CrossChainTransferability {
+	/// The asset is only transferable through XCM
+	Xcm,
+
+	/// The asset is only transferable through Centrifuge Connectors
+	Connectors,
+
+	/// The asset is transferable through all available options.
+	/// NOTE: Once set to `All`, the asset is automatically transferrable
```

Comment:
Maybe we should move/repeat part of this comment for the (missing) docs comment of the enum `CrossChainTransferability`.

Reply:
Done, let me me know how you find it now 👍